hr_employee_ppe/models/hr_employee_ppe.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Escodoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class HrEmployeePPE(models.Model):
+    """Adds PPE information and allocation."""
+
+    _name = 'hr.employee.ppe'
+    _description = 'Personal Protective Equipments'
+    _inherit = ['mail.thread', 'mail.activity.mixin']
+
+    name = fields.Many2one(
```

Comment:
I would call it equipment_id or something like this and then have another Char field name related to the name of the equipment if you want since it feels a bit weird to have a field called name that is a many2one.

Reply:
For me is better to rename it ppa_id, to avoid confusion with maintenance.equipment_id, and in general use ppa instead of equipment: ppa is a subcategory of equipments for me.

Reply:
it makes very sense the notes by @mymage

Reply:
OK, I agree, and maybe in the future there could be a maintenace module that links maintenace equipments with this.

Reply:
You are talking about PPEs as equipment but I think they could be simple stock products (like clothes) you want to know when they were delivered and renew them sometimes.
It doesn't mean that they could be an equipment, but a lot of times it should be managed as simple quants assigned to an employee.

At the ends equipments are elements of the company where we will create actions related to them, but a lot of PPEs only have 3 simple actions: purchase, assign to user, trash

Reply:
@etobella I agree, 90% of PPE are consumable, but in some case they have to be managed: ie radiation counter, anti gas mask. In this cases you can ad a reference to an equipment.

Reply:
I mean that maybe it will be necessary to manage ppe in two different ways, as consumables or as equipments, just that :wink:

Reply:
I share your thoughts, @etobella. Actually the goal is to create a simple (yet expandable) control of which employee has a valid PPE at certain moment, and be able to predict replacement needs.

Do you guys feel like it is important to link it to a product?

Reply:
(product link) Not important but useful, as consumable they might be used for some check/automation.
I don't know at this point if it is better to change name field in hr.employee.ppe.equipment with a ppe_product_id that point m2o to products.